tests/uitests/src/test/kotlin/ui/ComponentTestPreview.kt

```diff
@@ -28,4 +28,8 @@ class ComponentTestPreview(
     override val name: String = showkaseBrowserComponent.componentName
 
     override fun toString(): String = showkaseBrowserComponent.componentKey
+        // Strip package (Preview MUST all have a distinct name)
```

Comment:
Can we keep this as a last resort? There is noway to detect such clashes at compile time so it will degrade the devx.
Can we just strip the common `io.element.android` from the pacakge name for now?